py_gtfs_rt_ingestion/tests/test_s3_utils.py

```diff
@@ -87,11 +89,12 @@ def test_move_bad_objects(s3_stub, caplog):
     ]
     src_bucket = 'bad_src_bucket'
     dest_bucket = 'bad_dest_bucket'
-    
+
     caplog.set_level(logging.WARNING)
     with s3_stub:
-        move_3s_objects(bad_file_list, src_bucket, dest_bucket)
-    assert f"{src_bucket} bucket not found in bad_file1 path" in caplog.text
-    assert f"{src_bucket} bucket not found in bad_file2 path" in caplog.text
+        move_s3_objects(bad_file_list, dest_bucket)
+
+    for bad_file in bad_file_list:
+        assert "Unable to move %s to %s" % (bad_file, dest_bucket) in caplog.text
```

Comment:
had to change the test here.

now its looking for the error message, not the exception string.